src/App.tsx

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
-
 import './App.scss';
-import Start from './components/Start';
-import { Finish } from './components/Finish';
-
-import { isLoading, getMessage } from './store';
+import { CurrentUser } from './components/CurrentUser/CurrentUser';
+import { TodoList } from './components/TodoList/TodoList';
+import { getUserIdSelector } from './store/selectors';
+import './styles/general.scss';
 
-
-const App = () => {
-  const loading = useSelector(isLoading);
-  const message = useSelector(getMessage) || 'Ready!';
+const App: React.FC = () => {
+  const userId = useSelector(getUserIdSelector);
```

Comment:
no need to create a selector for userId. And store in in state
You can check if user exists when getting it in CurrentUser component